discretize/MeshIO.py

```diff
@@ -300,7 +300,56 @@ def _toVTRObj(mesh, models=None):
             vtkObj.GetCellData().SetActiveScalars(models.keys()[0])
         return vtkObj
 
-    def readModelUBC(mesh, fileName):
+    def _readModelUBC_2D(mesh, fileName):
+        """
+        Read UBC GIF 2DTensor model and generate 2D Tensor model in simpeg
+
+        Input:
+        :param string fileName: path to the UBC GIF 2D model file
+
+        Output:
+        :rtype: numpy.ndarray
+        :return: model with TensorMesh ordered
+        """
+
+        # Open fileand skip header... assume that we know the mesh already
+        obsfile = np.genfromtxt(
+            fileName, delimiter=' \n',
+            dtype=np.str, comments='!'
+        )
+
+        dim = np.array(obsfile[0].split(), dtype=int)
+
+        temp = np.array(obsfile[1].split(), dtype=float)
+
+        if len(temp) > 1:
+            model = np.zeros((dim[0], dim[1]))
+
+            for ii in range(len(obsfile)-1):
+                mm = np.array(obsfile[ii+1].split(), dtype=float)
+                model[:, ii] = mm
+
+            model = model[:, ::-1]
+
+        else:
+
+            if len(obsfile[1:]) == 1:
```

Comment:
would you mind adding a test that hits these lines?

Reply:
we can actually make this function much simpler. Will do.